lib/libvarnishapi/vut.c

```diff
@@ -378,7 +378,6 @@ VUT_Fini(struct VUT **vutp)
 	if (vut->vsm)
 		VSM_Destroy(&vut->vsm);
 
-	memset(vut, 0, sizeof *vut);
```

Comment:
Why remove this one? I know it has a high chance of being optimized out, but why remove it?